components/blitz/resources/omero/FS.ice

```diff
@@ -398,6 +399,7 @@ module omero {
                                 int blockSize,
                                 bool ignoreSysFiles,
                                 bool ignoreDirEvents,
+                                bool platformCheck,
```

Comment:
For 5.1, this makes sense, but for backporting to 5.0 we'll have to evaluate breakages.